.bazelrc

```diff
@@ -48,7 +48,7 @@ run:remote_caching_ro --remote_download_toplevel
 
 # Import the bazel caching config, with default disk cache,
 # which in CI gets overwritten by the remote caching config. 
-import bazel/bazelrcs/cache.bazelrc
+import /var/cache/cache.bazelrc
```

Comment:
We could also use `import %workspace%/bazel/bazelrcs/cache.bazelrc`, see https://docs.bazel.build/versions/main/guide.html#imports

Reply:
This is a really clean solution, the other PR setting up symbolic links is not even needed.

Reply:
Is it possible to replace all links by `%workspace%/.bazel-cache/` etc?

Reply:
I tried `%workspace%/.bazel-cache/` and it works fine.

Reply:
Thanks, good to know. Then we might refactor the mechanics at some point to get rid of all the linking. But for now let us just keep it as it is as this is a larger refactoring.

Reply:
I stand corrected:
![workspace_not_working_bazel_cache](https://user-images.githubusercontent.com/99820677/183646599-0a121581-cdef-455f-9d6c-a1258051d888.png)
But it might work with a little more thought put into it.